src/acquisition/covidcast/csv_importer.py

```diff
@@ -120,13 +120,13 @@ def find_csv_files(scan_dir, issue=(date.today(), epi.Week.fromdate(date.today()
         # safe to ignore this file
         continue
 
-      logger.info('file:', path)
+      logger.info(event='file:', file=path)
```

Comment:
"file:" is too generic. what's the purpose of this message, is it just marking where handling for `path` begins?